src/datasources/entity-ds/datasource.js

```diff
@@ -361,7 +361,12 @@ export class OpenNMSEntityDatasource {
     }
 
     clearAlarm(alarmId) {
```

Comment:
The escalate, acknowledge and un-acknowledge actions (available via the right click context menu) should also be updated to function similarly.

Reply:
Other actions working in similar pattern because `performAlarmActionOnDatasource` is common for them. `clearAlarm` doesn't need to handle promise necessarily.. updated the PR in this commit https://github.com/OpenNMS/opennms-helm/pull/254/commits/622a8e48a7860dfa4305cdf0810952919bb74e7a.
Thanks!! :)